src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write code here
+const headElements = document
+  .querySelector('thead')
+  .querySelectorAll('th');
+
+for (const item of headElements) {
+  item.addEventListener('click', () => {
+    const table
+      = [...document.querySelector('tbody').querySelectorAll('tr')]
+        .sort(compareValues);
+
+    function compareValues(aInput, bInput) {
+      let a = aInput.children[item.cellIndex].textContent;
+      let b = bInput.children[item.cellIndex].textContent;
+
+      if (convert(a) > 0) {
+        a = (convert(a));
+        b = (convert(b));
+      }
+
+      return a < b ? -1 : 1;
+    }
```

Comment:
It looks like this function it not belong to here as well. move it on global level

Reply:
I use 'item.cellIndex' inside loop and it must be inside the loop.